src/test/java/de/tum/in/www1/artemis/exam/ExamIntegrationTest.java

```diff
@@ -689,49 +209,6 @@ void testGenerateStudentExams() throws Exception {
             assertThat(studentExam.getExam()).isEqualTo(exam);
             // TODO: check exercise configuration, each mandatory exercise group has to appear, one optional exercise should appear
         }
-
-        // Make sure delete also works if so many objects have been created before
-        request.delete("/api/courses/" + course1.getId() + "/exams/" + exam.getId(), HttpStatus.OK);
-    }
-
-    @Test
-    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
-    void testGenerateStudentExamsCleanupOldParticipations() throws Exception {
-        Exam exam = examUtilService.setupExamWithExerciseGroupsExercisesRegisteredStudents(TEST_PREFIX, course1, NUMBER_OF_STUDENTS);
-
-        request.postListWithResponseBody("/api/courses/" + course1.getId() + "/exams/" + exam.getId() + "/generate-student-exams", Optional.empty(), StudentExam.class,
-                HttpStatus.OK);
-
-        List<Participation> studentParticipations = participationTestRepository.findByExercise_ExerciseGroup_Exam_Id(exam.getId());
-        assertThat(studentParticipations).isEmpty();
-
-        // invoke start exercises
-        studentExamService.startExercises(exam.getId()).join();
-
-        studentParticipations = participationTestRepository.findByExercise_ExerciseGroup_Exam_Id(exam.getId());
-        assertThat(studentParticipations).hasSize(12);
-
-        request.postListWithResponseBody("/api/courses/" + course1.getId() + "/exams/" + exam.getId() + "/generate-student-exams", Optional.empty(), StudentExam.class,
-                HttpStatus.OK);
-
-        studentParticipations = participationTestRepository.findByExercise_ExerciseGroup_Exam_Id(exam.getId());
-        assertThat(studentParticipations).isEmpty();
-
-        // invoke start exercises
-        studentExamService.startExercises(exam.getId()).join();
-
-        studentParticipations = participationTestRepository.findByExercise_ExerciseGroup_Exam_Id(exam.getId());
-        assertThat(studentParticipations).hasSize(12);
-
-        // Make sure delete also works if so many objects have been created before
-        request.delete("/api/courses/" + course1.getId() + "/exams/" + exam.getId(), HttpStatus.OK);
```

Comment:
This is missing in the corresponding function in `ExamParticipationIntegrationTest`. Is it on purpose?

Reply:
good catch, it was by accident :sweat_smile: